lib/bugcrowd_templates.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require_relative 'bugcrowd_templates/version'
+require 'pathname'
+require 'bugcrowd_templates/template_path'
+
+module BugcrowdTemplates
+  class TemplateNotFoundError < StandardError; end
+
+  DATA_DIR = Pathname.new(Gem::Specification.find_by_name('bugcrowd_templates').gem_dir)
+
+  # if we need to add new template type
+  # we can add here
+  TEMPLATE_TYPES = {
+    submissions: 'submissions',
+    methodology: 'methodology'
+  }.freeze
+
+  module_function
+
+  # returns the Bugcrowd template based on given inputs
+  def get(
+    template_type: '',
```

Comment:
Is `template` here necessary? Seems redundant, given you're already calling a module named `BugcrowdTemplates`

Reply:
Yes, i have removed `template` in arguments, now we can call the below way
`BugcrowdTemplates.get(type: 'methodology', field: 'notes', category: 'website_testing', subcategory: 'information')
`